src/scripts/components/tabs.js

```diff
@@ -0,0 +1,72 @@
+const tablistWrappers = document.querySelectorAll('[data-pos-component="Tabs"]');
+tablistWrappers.forEach(tablistWrapper => {
+  const tabs = [...tablistWrapper.querySelectorAll('button')];
```

Comment:
what if there is a button in the tab panel content? we should use a data attribute here instead